content/en/docs/tasks/tools/included/verify-kubectl.md

```diff
@@ -12,7 +12,7 @@ In order for kubectl to find and access a Kubernetes cluster, it needs a
 [kubeconfig file](/docs/concepts/configuration/organize-cluster-access-kubeconfig/),
 which is created automatically when you create a cluster using
 [kube-up.sh](https://github.com/kubernetes/kubernetes/blob/master/cluster/kube-up.sh)
-or successfully deploy a Minikube cluster.
+or successfully deploy a [Minikube](https://minikube.sigs.k8s.io/docs/start/) cluster.
```

Comment:
Hi @deadandhallowed, you can refer to this [comment](https://github.com/kubernetes/website/issues/44981#issuecomment-1922102046) for further changes.

Reply:
```suggestion
or successfully deploy a {{<glossary_tooltip text="Minikube" term_id="minikube" >}} cluster.
```